Core/include/Acts/Utilities/KDTree.hpp

```diff
@@ -318,369 +310,249 @@ class KDTree {
   /// A k-d tree consists of two different node types: leaf nodes and inner
   /// nodes. These nodes have some common functionality, which is captured by
   /// this common parent node type.
-  class KDTreeAbstractNode {
+  class KDTreeNode {
    public:
+    /// @brief Enumeration type for the possible node types (internal and leaf).
+    enum class NodeType { INTERNAL, LEAF };
```

Comment:
Existing style would favor:
```suggestion
    enum class NodeType { Internal, Leaf };
```

Reply:
Fixed.